ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -531,7 +597,32 @@ func convertToTaskDefinitionInTest(t *testing.T, name string, serviceConfig *con
 		EnvironmentLookup: envLookup,
 		ResourceLookup:    resourceLookup,
 	}
-	taskDefinition, err := ConvertToTaskDefinition(taskDefName, context, serviceConfigs, taskRoleArn)
+	taskDefinition, err := ConvertToTaskDefinition(taskDefName, context, serviceConfigs, taskRoleArn, "")
+	if err != nil {
+		t.Errorf("Expected to convert [%v] serviceConfigs without errors. But got [%v]", serviceConfig, err)
+	}
+	return taskDefinition
+}
+
+func convertToTaskDefWithEcsFieldsInTest(t *testing.T, name string, serviceConfig *config.ServiceConfig, taskRoleArn string, ecsFieldsFileName string) *ecs.TaskDefinition {
+	serviceConfigs := config.NewServiceConfigs()
+	serviceConfigs.Add(name, serviceConfig)
+
+	taskDefName := "ProjectName"
+	envLookup, err := GetDefaultEnvironmentLookup()
+	if err != nil {
+		t.Fatal("Unexpected error setting up environment lookup")
+	}
+	resourceLookup, err := GetDefaultResourceLookup()
+	if err != nil {
+		t.Fatal("Unexpected error setting up resource lookup")
+	}
+	context := &project.Context{
+		Project:           &project.Project{},
+		EnvironmentLookup: envLookup,
+		ResourceLookup:    resourceLookup,
+	}
+	taskDefinition, err := ConvertToTaskDefinition(taskDefName, context, serviceConfigs, taskRoleArn, ecsFieldsFileName)
 	if err != nil {
 		t.Errorf("Expected to convert [%v] serviceConfigs without errors. But got [%v]", serviceConfig, err)
```

Comment:
Use assert

Reply:
no, I actually want to return the error here.